api/src/modules/projects/projects.service.ts

```diff
@@ -21,14 +26,18 @@ export class ProjectsService extends AppBaseService<
 
   async extendFindAllQuery(
     query: SelectQueryBuilder<Project>,
-    fetchSpecification: FetchSpecification,
+    fetchSpecification: ProjectFetchSpecificacion,
   ): Promise<SelectQueryBuilder<Project>> {
+    const filter = fetchSpecification.filter.projectName
+      .map((name) => `'%${name}%'`)
+      .join(',');
     // Filter by project name
     if (fetchSpecification?.filter?.projectName) {
-      query = query.andWhere('project_name ILIKE :projectName', {
-        projectName: `%${fetchSpecification.filter.projectName}%`,
+      query = query.andWhere('project_name ILIKE ANY(ARRAY[:projectName])', {
```

Comment:
This is an attemp to "override" the behavior of the library so that we can:
1. keep the interface type, each property holding an array of values
2. use an existing property to filter by, otherwise, if we keep the validating zod schema, we would need a different query property for this